_docs-sources/foundations/grunty/architecture.mdx

```diff
@@ -0,0 +1,94 @@
+# Architecture
+
+The Grunty GitHub App is a GitHub App introduced to help reduce the burden of integrating Gruntwork products to GitHub resources. The app is designed to be lightweight and flexible, providing a simple way to get started with Gruntwork products.
```

Comment:
This section is framed as the architecture behind the _GitHub App,_ but what I'd rather understand is the overall architecture:

1. Developer portal
2. GitHub app
3. GitHub Actions workflows

And how those interact. Perhaps it's better to have an architecture page in the "Intro" section, and a dedicated page for GitHub App permissions, which I think is what you're mostly covering here.